docs/reference/terminal49/terminal49.v1.json

```diff
@@ -5950,6 +5957,11 @@
                 "type": "string",
                 "description": "IANA tz. Applies to attribute empty_terminated_at.",
                 "nullable": true
+              },
+              "pickup_rail_carrier_scac": {
```

Comment:
It wasn't clear from the serializer to me if the value is nullable of the whole attribute is optional?

Reply:
is nullable and the attribute is only present if you are in the beta